paddle/majel/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(MAJEL_CXX_FILES place.cpp)
```

Comment:
> If CMake detects the host machine equipped GPUs, it will invoke cuda_add_library to generate the library, vice visa.

If we are going to port Majel to Paddle, I think we will have to enable GPU and CPU versions. Also, we'll have to be able to specify which one to build, as we do with Paddle.  Also, I think we shouldn't assume that the development computer having a GPU means that the runtime environment would have GPUs.